modules/doc/content/newsletter/2022_03.md

```diff
@@ -5,14 +5,101 @@ This MOOSE Newsletter edition is in progress. Please check back in April 2022
 for a complete description of all MOOSE changes.
 !alert-end!
 
+## MPICH Updated to 4.0.1, GCC to 10.3.0
+
+The `moose-mpich` package has been updated to 4.0.1 and now relies entirely on
+the MPICH built by the conda-forge maintainers (`mpich-mpicc`, `mpich-mpicxx`,
+and `mpich-mpifort`). As a result of this change, the default compilers installed
+on Linux using the MOOSE package are now GCC 10.3.0 / gfortran 10.3.0. Minimum
+supported compilers (see [below](2022_03.md#cxx17-support)) remain the same and
+Clang 12.0.1 / gfortran 9.3.0 is still the default compiler set on MacOS. The bulk
+of MOOSE PR testing will still be done using GCC 9, to maintain compatibility over
+a broad range of compiler releases.
+
+To update your MOOSE packages, please activate your MOOSE conda environment and
+perform the command:
+
+```
+conda update --all
+```
+
+The following package versions and builds outlined in the update prompt should be
+at least the following:
+
+```
+Package                  Version            Build
+=====================================================
+moose-libmesh            2022.03.18         build_0
+moose-libmesh-vtk        9.1.0              build_5
+moose-petsc              3.16.5             build_0
+moose-mpich              4.0.1              build_0
+```
+
 ## MOOSE Improvements
 
-### C++17 Support
+### C++17 Support id=cxx17-support
+
+MOOSE now requires a C++17 capable compiler and supports C++17 features. With
+this, GCC 7 and Clang 5 are now the minimum supported compilers.
+
+### HDF5 support in PETSc and libMesh now requested by default
+
+The `update_and_rebuild_petsc` script now performs a basic search for a supported
+HDF5 installation in the local environment by default. Developers can place an
+HDF5 installation in the following searched locations:
+
+- `/usr/lib/hdf5`
+- `/usr/local/hdf5`
+- `/usr/share/hdf5`
+- `/usr/local/hdf5/share`
+- `$HOME/.local`
+
+Or select a custom location by setting one of the environment variables `HDF5_DIR`,
+`HDF5DIR`, or `HDF5_ROOT`. If a compatible installation cannot be found, PETSc
+will download and build it for the user on Linux. This fallback download option
+is not yet available on MacOS, but will be coming soon. Subsequently, the
+`update_and_rebuild_libmesh` script now uses the `--enable-hdf5` configure flag
+and, with recent libMesh changes, attempts to find the HDF5 installation used by
+PETSc.
 
-MOOSE now requires a C++17 capable compiler and supports C++17 features. With this, GCC 7 and Clang 5 are now the minimum supported compilers.
+The new conda packages mentioned above all contain the PETSc default HDF5 on Linux,
+but no HDF5 is currently provided on MacOS (coming soon). A serial, pre-built HDF5
+is available within the `moose-tools` package on either platform if this is sufficient
+for developer needs.
 
 ## libMesh-level Changes
 
+### `2022.03.18` Update
```

Comment:
Most of this content should be ripped out as I'd prefer that the newsletter not be a changelog; ping @roystgnr and @lindsayad for anything important

Reply:
We do add this information regularly whenever libMesh is updated though.

Reply:
And I only added a portion of this myself. Vast majority of it came from @lindsayad

Reply:
Yes and things should only be in it if they're readable without context; right know the majority of it is not

Reply:
Eh that's fair. I'll revise it

Reply:
I’m all for information but people will ignore it in time if it’s not understandable 😬